tools/zetasql-helper/README.md

```diff
@@ -1,172 +1,9 @@
-# BigQuery SQL Analyzer
+# ZetaSQL Helper
 
-This project is meant to be use as a starting point for developers interested in
-analyzing BigQuery SQL with [ZetaSQL](https://github.com/google/zetasql).
+This project is currently a work in progress that will provide tools for
```

Comment:
I would prefer if we directly pointed to the zetasql-toolkit

Reply:
Done!